### ISSUE TITLE
Support CFN template files larger than 51200

### ISSUE THREAD

**Issue by rene84** [org-formation/org-formation-cli#67]
***Expected behaviour***
When including template files larger than 51200 bytes, OrgFormation should not throw an error, but instead recognize that the way to process larger templates is to first upload them to S3 and reference the S3 object in the CFN command

***Current behaviour***
```
}' at 'templateBody' failed to satisfy constraint: Member must have length less than or equal to 51200
ERROR: task **** failed, reason: number failed stacks 1 exceeded tolerance for failed stacks 0
number failed stacks 1 exceeded tolerance for failed stacks 0
Error: number failed stacks 1 exceeded tolerance for failed stacks 0
    at Object.onFailureToleranceExceeded (/usr/local/lib/node_modules/aws-organization-formation/dist/src/cfn-binder/cfn-task-runner.js:24:23)
    at Function.RunTasks (/usr/local/lib/node_modules/aws-organization-formation/dist/src/core/generic-task-runner.js:49:26)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Function.RunTasks (/usr/local/lib/node_modules/aws-organization-formation/dist/src/cfn-binder/cfn-task-runner.js:29:9)
    at async UpdateStacksCommand.performCommand (/usr/local/lib/node_modules/aws-organization-formation/dist/src/commands/update-stacks.js:77:17)
    at async Function.Perform (/usr/local/lib/node_modules/aws-organization-formation/dist/src/commands/update-stacks.js:15:9)
    at async Object.perform (/usr/local/lib/node_modules/aws-organization-formation/dist/src/build-tasks/tasks/update-stacks-task.js:27:17)
    at async Function.performTask (/usr/local/lib/node_modules/aws-organization-formation/dist/src/core/generic-task-runner.js:65:17)
    at async Promise.all (index 0)
    at async Function.RunTasks (/usr/local/lib/node_modules/aws-organization-formation/dist/src/core/generic-task-runner.js:43:13)
ERROR: number failed tasks 1 exceeded tolerance for failed tasks 0
```

***Way to reproduce***
Create a tasks file larger than 51200 bytes and process that using a tasksfile.yml that includes that template in the update-stacks command

**Comment by OlafConijn** [comment]
awesome suggestion! thanks! @rene84

**Comment by OlafConijn** [comment]
fixed in v0.9.12
